tests/upgrade/post/go.mod

```diff
@@ -0,0 +1,202 @@
+module github.com/oasisprotocol/oasis-core/test-upgrade
+
+go 1.19
+
+replace (
+	// Updates the version used by badgerdb, because some of the Go
+	// module caches apparently have a messed up copy that causes
+	// build failures.
+	// https://github.com/google/flatbuffers/issues/6466
+	github.com/google/flatbuffers => github.com/google/flatbuffers v1.12.1
+
+	// v1.5.0 has broken uint parsing, use my commit with fixes instead until
+	// the maintainers merge my PR: https://github.com/spf13/cast/pull/144
+	github.com/spf13/cast => github.com/oasisprotocol/cast v0.0.0-20220606122631-eba453e69641
+
+	github.com/tendermint/tendermint => github.com/oasisprotocol/tendermint v0.34.23-oasis2
+
+	// Required because of https://github.com/libp2p/go-libp2p-pubsub/issues/467.
+	github.com/whyrusleeping/timecache => github.com/oasisprotocol/timecache v0.0.0-20220102191729-558b1c931038
```

Comment:
Unrelated to this PR, but this can be removed (also from master) as go-libp2p-pubsub 0.9.3 no longer uses this.